### DISCUSSION TITLE
Order events by weekday

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [solspace/craft-calendar#84] [Feature Requests]
How can i order all events by weekday? Like `.orderBy('weekdayDate ASC')`

**Comment by kjmartens** [comment]
This isn't possible at this time, unfortunately, but we'll add it to the feature requests list for future consideration. 🙂 

You can otherwise consider using the [calendar.week function](https://docs.solspace.com/craft/calendar/v3/template-functions/calendar.week.html) if your intent is to just display 1 week's event results at a time.

**Comment by alexanderbuergin** [comment]
That option would be nice, if you add it to the list :)
